### PR TITLE
Week5

### PR DESCRIPTION
**Pull request by AnnaPestova1**

### CODE REVIEW

File: 03-task-manager/optionalArrayMethodsReviewExtraAssignment.js [hunk 1]
```diff
@@ -154,6 +154,86 @@ const names = [
...
+upper.push(el[0].toUpperCase())
+  }
+const lower = el.slice(1)
+const nameString = upper.toString()+lower.toLowerCase()
+nameFormat.push(nameString)
+})
+return nameFormat.join(" ")
+})
+const lastNameEndsZ = titleCase.filter((person)=>{
+const p=person.split(" ")
```
Comment (joseluiz) [inline 03-task-manager/optionalArrayMethodsReviewExtraAssignment.js:229]: Suggestion: You don't need to split the person's name to check if it ends with z, you can just use person.at(-1), and it should work.

File: 03-task-manager/starter/controllers/tasks.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+const getAllTasks = (req, res)=>{
+    res.send("get all tasks")
+}
+
+const createTask = async (req, res)=>{
+    const task = await Task.create(req.body)
+    res.status(200).json({task})
+}
+
```
Comment (joseluiz) [inline 03-task-manager/starter/controllers/tasks.js:11]: You need to implement these methods retrieving and updating data on MongoDB. See the example located in the final/controller directory to know how to implement this.

**Review by joseluiz — CHANGES_REQUESTED** [review]
You only implemented the method create task using the MongoDB to save the data, you need to implement the other methods to save and retrieve data from the MongoDB.
If you have any questions please call me on Slack.

File: 03-task-manager/optionalArrayMethodsReviewExtraAssignment.js (hunk 1 above)
Comment (AnnaPestova1) [reply 03-task-manager/optionalArrayMethodsReviewExtraAssignment.js:229]: That's right. Thanks!

File: 03-task-manager/starter/controllers/tasks.js (hunk 2 above)
Comment (AnnaPestova1) [reply 03-task-manager/starter/controllers/tasks.js:11]: retrieving and updating data on MongoDB - it's the next week assignment :)

Comment (joseluiz) [reply 03-task-manager/starter/controllers/tasks.js:11]: Ok, sorry, I thought every CRUD operation was required.

**Review by joseluiz — APPROVED** [review]
PR approved.

**Pull request merged**